en/setup/dev_env_linux.md

```diff
@@ -128,21 +128,45 @@ Note: Alternatively, especially on Mac, you can also use [nano-dm](https://githu
 
 ### Raspberry Pi hardware
 
-Developers working on Raspberry Pi hardware should download the RPi Linux toolchain from below. The installation script will automatically install the cross-compiler toolchain. If you are looking for the _native_ Raspberry Pi toolchain to compile directly on the Pi, see [here](../flight_controller/raspberry_pi.md#native-builds-optional)
+Developers working on Raspberry Pi hardware need to download a ARMv7 cross-compiler, either GCC or clang.
+The recommended toolchain for raspbian is GCC 4.8.3 and can be cloned from `https://github.com/raspberrypi/tools.git`.
+The `PATH` environmental variable should include the path to the gcc cross-compiler collection of tools (e.g. gcc, g++, strip) prefixed with `arm-linux-gnueabihf-`.
 
 ```sh
-git clone https://github.com/pixhawk/rpi_toolchain.git
-cd rpi_toolchain
-./install_cross.sh
-```
+git clone https://github.com/raspberrypi/tools.git ${HOME}/aero-workspace/rpi-tools
 
-You will be required to enter your password for toolchain installation to complete successfully.
+# test compiler
+$HOME/aero-workspace/rpi-tools/arm-bcm2708/gcc-linaro-arm-linux-gnueabihf-raspbian-x64/bin/arm-linux-gnueabihf-gcc -v
 
-You can pass a different path to the installer script if you wouldn't like to install the toolchain to the default location of `/opt/rpi_toolchain`. Run `./install_cross.sh <PATH>`. The installer will automatically configure required environment variables as well.
+# permanently update PATH variable by modifying ~/.profile
+echo 'export PATH=$PATH:$HOME/aero-workspace/rpi-tools/arm-bcm2708/gcc-linaro-arm-linux-gnueabihf-raspbian-x64/bin/arm-linux-gnueabihf-' >> ~/.profile
```

Comment:
Shouldn't this be `echo 'export PATH=$PATH:$HOME/aero-workspace/rpi-tools/arm-bcm2708/gcc-linaro-arm-linux-gnueabihf-raspbian-x64/bin' >> ~/.profile`?